code/code/cmd/cmd_stat.cc

```diff
@@ -1721,6 +1721,7 @@ void TBeing::statBeing(TBeing *k)
       case SPELL_CURE_POISON_DEIKHAN:
       case SKILL_DISARM_DEIKHAN:
       case SPELL_HEAL_CRITICAL_DEIKHAN:
+      case SPELL_MARTIAL_MIGHT:
```

Comment:
This is a case statement with a bunch of cases here that follow the same logic below. The stat command seems to work. I obviously did test all possible combinations but I really don't understand how it could be a problem. There's another enum I changed that I was/am scared about again.

Reply:
Oof, I misread!